src/server.ts

```diff
@@ -388,14 +392,18 @@ export class Server {
     // is an ScSymbol and the last is a U32.
     //
     // The difficulty comes in matching up the correct integer primitives.
-    return jsonrpc.postObject(this.serverURL.toString(), "getEvents", {
-      filters: request.filters ?? [],
-      pagination: {
-        ...(request.cursor && { cursor: request.cursor }), // add if defined
-        ...(request.limit && { limit: request.limit }),
-      },
-      ...(request.startLedger && { startLedger: request.startLedger.toString() }),
-    });
+    return jsonrpc.postObject<SorobanRpc.RawGetEventsResponse>(
```

Comment:
is there a reason why `_getEventsRaw()` is not used here, following same 'pair of methods' pattern for raw and encoded responses?

Reply:
nope, just an oversight. thanks for the catch!